src/MAUI/Maui.Samples/Samples/UtilityNetwork/PerformValveIsolationTrace/PerformValveIsolationTrace.xaml.cs

```diff
@@ -283,6 +273,10 @@ private async void OnGeoViewTapped(object sender, Esri.ArcGISRuntime.Maui.GeoVie
                 Graphic traceLocationGraphic = new Graphic(feature.Geometry as MapPoint ?? e.Location, symbol);
                 _barrierOverlay.Graphics.Add(traceLocationGraphic);
             }
+            catch (OperationCanceledException ex)
```

Comment:
I'm not sure that we just want to catch and consume this here?

Reply:
This is to allow users to tap outside the bounds of the pop-up DisplayActionSheet (especially buttons in other UI elements) without triggering an unhandled invalid value exception; this way, the DisplayActionSheet can be cancelled with no further action taken